R/bucket_list.R

```diff
@@ -61,7 +74,9 @@ bucket_list <- function(
   group_name,
   group_put_max = rep(Inf, length(labels)),
   options = sortable_options(),
-  class = "default-sortable"
+  class = "default-sortable",
+  orientation = c("horizontal", "vertical"),
+  dependencies = NULL
```

Comment:
let's remove this parameter.

Reply:
Agreed. I'm removing the `dependencies` argument and will create instructions for downstream users to add their own CSS.